bindings/python/eblob_python.cpp

```diff
@@ -52,6 +53,29 @@ static void eblob_extract_id(const struct eblob_id &e, struct eblob_key &id)
 	eblob_extract_arr(e.id, id.id, &len);
 }
 
+/**
+ * __eblob_read_ll() - interruption-safe wrapper for pread(2)
+ */
+static int __eblob_read_ll(int fd, char *data, size_t size, off_t offset)
```

Comment:
Client code can be interrupted by various reasons and the main one is usually to stop processing.
There are no dedicated signal handlers in client code (compared to server one) except ignoring some signals in workers, which brings me to this question - what will prevent client code from running forever ignoring signals, since it returns back into syscall right after it has ignored signal?

At the server-side there is no such problem, since there is a signal handler which will set exiting flags, and thread which has been interrupted will return to higher level and eventuall will check this flag, and thus will exit.